.hound.yml

```diff
@@ -2,4 +2,4 @@
 
 eslint:
   enabled: true
-  config_file: ".eslintrc.json"
+  config_file: .eslintrc
```

Comment:
You tried to correct this line but you should not remove the extension, it should be like this `config_file: .eslintrc.json`

Reply:
Yeah I am looking on that